RecoBTag/SoftLepton/src/MuonTagger.cc

```diff
@@ -35,8 +36,8 @@ float MuonTagger::discriminator(const TagInfoHelper& tagInfo) const {
     bool flip(false);
     if(m_selector.isNegative()) {
       int seed=1+round(10000.*properties.deltaR);
```

Comment:
Yes, I confirm.
The RECO algorithms should in general not contain any randomness.
The best is if the random generator is not used at all, but in cases where it's used the algorithm seed is event/object-specific so that the results are reproducible.